config/config.go

```diff
@@ -15,12 +15,29 @@ import (
 // PresetKey is the Viper config key to use in [viper.Viper.Set].
 const PresetKey = "preset"
 
+const (
+	PagingAlways Paging = "always"
+	PagingNever  Paging = "never"
+)
+
+// PagingOptions can be used for validation and schema generation (not implemented)
+var (
+	PagingOptions []Paging = []Paging{
+		PagingAlways,
+		PagingNever,
+	}
+)
+
+type Paging string
+
 type Config struct {
 	Debug             bool          `jsonschema:"-"`
 	Kubectl           string        `jsonschema:"default=kubectl,example=kubectl1.19,example=oc"` // Which kubectl executable to use
 	ObjFreshThreshold time.Duration // Ages below this uses theme.data.durationfresh coloring
 	Preset            Preset        // Color theme preset
-	Theme             Theme
+	Theme             Theme         //
+	Pager             string        // Command to use as pager
```

Comment:
I think `Pager` should be `*string` to allow null value.

There's a distinction from not setting the PAGER, vs setting PAGER to empty string.

Not setting it should result in using the default value. Setting it to an empty string should in disabling the pager.

Reply:
@applejag As I understood your very last proposal we would use the `paging` option to enable and disable the proposal. Does this mean you want the user to also be able to disable paging by unsetting the pager? (Currently not setting and setting to an empty string both leads to using `defaultPager()` and does not affect whether paging is enabled.)

I'm also not sure how the distinction between not setting and setting to an empty value is going to work, since unset environment variables are empty strings. Is this something that can only be done in the config file?

Reply:
> I'm also not sure how the distinction between not setting and setting to an empty value is going to work, since unset environment variables are empty strings.
I was wrong about that! Go's `os.Getenv()` doesn't distinguish between empty and unset, but Viper uses a different function that does.

I'm not sure though what the added benefit is of allowing something like `KUBECOLOR_PAGER=` to disable paging over the two alternatives we already have, i.e. `KUBECOLOR_PAGING=` and `--no-paging`. It should also be documented if unset and empty behave differently.

Reply:
Yeah Go has [`os.LookupEnv`](https://pkg.go.dev/os#LookupEnv), and then `os.Getenv` as a utility function when you don't care about the distinction between unset vs empty.

In bash the difference would be:

```bash
unset KUBECOLOR_PAGER
# vs
export KUBECOLOR_PAGER=""
```

But you're right. I was mixing Git's approach with Bat's approach a bit too much. Since we have `KUBECOLOR_PAGING`, then we don't need another layer of distinction between empty vs unset.